modules/preparing-an-inital-cluster-deployment-for-mce-disconnected.adoc

```diff
@@ -0,0 +1,130 @@
+// Module included in the following assemblies:
+//
+// * installing_with_agent_based_installer/preparing-an-agent-based-installed-cluster-for-mce.adoc
+
+:_content-type: PROCEDURE
+[id="preparing-an-inital-cluster-deployment-for-mce-disconnected_{context}"]
+
+= Preparing an initial cluster deployment for multicluster engine for Kubernetes(MCE) Operator while disconnected
+
+You can mirror the required {product-title} container images, the multicluster engine for Kubernetes(MCE) Operator, and Local Storage Operator (LSO) into your local mirror registry in a disconnected environment.
+Ensure that you note the local DNS and port of your mirror registry.
+
+[NOTE]
+====
+To mirror your {product-title} image repository to your mirror registry, you can use either the `oc adm release image`
+or `oc mirror` command. In this procedure, the `oc mirror` command is used as an example.
+====
+
+.Procedure
+
+. Create a `<assets_directory>` folder to place a valid `install-config.yaml` and `agent-config.yaml` file. This directory is useed to store all the assets.
```

Comment:
@andfasano  here we define the assets directory to store both the yaml files. Will that suffice?